cornellh4i/ctw-admin
# Implemented mesh type selector component

## Overview

Closes #30 : Create a collapsible multi-select component for the mesh type selector as shown in the midfis

## Changes Made

The component has attribute typeList: type: Array<string>, a list of the mesh types options than can be selected in the dropdown. The current selected mesh(es) are stored in state variable selected.

## Test Coverage

Note: the component is created and working but it is not used/placed in a page at this time. To test, use the MeshSelector component in a random page (e.g. admin) and remove after testing.
1) git checkout branch mesh-selector-component
2) import MeshSelector into the admin page (for testing purposes only, remove after testing)
3) copy <MeshSelector typeList={["Raschel", "Stainless", "3D Net"]}/> into the admin page (placeholder mesh type options)
4) interact with component at localhost:3000/admin
5) (optional) add  <p>{selected}</p> to the end of MeshSelector.tsx code to ensure the current selected meshes are correctly stored in state variable selected.
